src/example/heat_equation_two_layers/heat_equation_two_layers.jl

```diff
@@ -0,0 +1,168 @@
+module HeatEquationTwoLayers #hide
+println("Running heat equation two layers example...") #hide
+# # Heat equation
+# # Theory
+# This example shows how to solve the heat equation with eventually variable physical properties in steady and unsteady formulations:
+# ```math
+#   \rho C_p \partial_t u - \nabla . ( \lambda u) = f
```

Comment:
Small typo (probably coming from the legacy file), there is a \nabla missing before u.

Reply:
fixed